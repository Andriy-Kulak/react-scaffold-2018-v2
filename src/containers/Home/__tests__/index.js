/* eslint-env node, jest */

import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

import { Home } from '../index';

Enzyme.configure({ adapter: new Adapter() });

const actions = {
  getAPIData: () => {
  }
};

describe('>>> Home Standard Render Container ~~ Snapshot ~~', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(<Home actions={actions} />);
    expect(toJson(renderedValue)).toMatchSnapshot();
  });
});