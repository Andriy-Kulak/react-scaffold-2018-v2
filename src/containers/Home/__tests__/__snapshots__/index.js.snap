// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Home Standard Render Container ~~ Snapshot ~~ +++capturing Snapshot with 1`] = `
<Layout>
  <styled.div>
    <styled.div>
      <h2>
        Welcome to React
      </h2>
    </styled.div>
    <p
      className="app-intro"
    >
      To get started, edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <styled.div
      className="app-intro"
    >
      Your IP is: 
    </styled.div>
  </styled.div>
</Layout>
`;
